packages/round-manager/docs/DEV.md

```diff
@@ -15,6 +15,15 @@ Create environment files, and fill in environment variables with your own values
 cp ../.env.sample ../.env
 ```
 
+The following may be helpful when filling in the the environment variables.
+
+For `REACT_APP_PINATA_JWT` and `REACT_APP_PINATA_GATEWAY`, contact @thelostone-mc for credentials 
```

Comment:
I was not sure what to put for the following credentials. @thelostone-mc lmk is this sounds fine or if it should be changed

Reply:
I would say let's update it to set up your PINATA account. They have a free service 🙌🏼

Reply:
Ok, fixed it